src/api/app/models/project.rb

```diff
@@ -1245,14 +1246,34 @@ def update_product_autopackages
   end
 
   def open_requests
-    reviews = BsRequest.collection(project: name, states: %w(review)).map(&:number)
-    targets = BsRequest.collection(project: name, states: %w(new)).map(&:number)
-    incidents = BsRequest.collection(project: name, states: %w(new), types: %w(maintenance_incident)).map(&:number)
-
+    reviews = BsRequest.with_involved_projects(id)
```

Comment:
Arrrgh!, `reviews = BsRequest.with_involved_projects(id)...` is faster than before, but the next two are worse, they are slower :sob:.